src/routers/router.ts

```diff
@@ -1,14 +1,41 @@
+import Koa from "koa"
 import Router from "koa-router"
 import { sum } from "../services/sum"
 
+export let users = [{

```

Comment:
perché è esportato? lo usi da qualche altra parte (ps comunque semmai l'avrei spostato in un altro file, non nel router)

Reply:
perché volevo usarlo nel test, usarlo nel beforeEach del test e poi ho dimenticato di cancellare